### PR TITLE
Fix transaction statistics - Closes #538

### LINKED ISSUE #538: Transaction statistics calculation update
### Actual behavior

There is a bug that does not show the sum of all transactions per type.

### Expected behavior

Make sure all transactions per type are calculated properly.

### Steps to reproduce

1. Start the service on the betanet.
2. Check the results.

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #538 

### How was it solved?
- [x] Update logic to generate correct primary id
- [x] set `forceReload` to `true` during `init`
- [x] Update Jenkins to remove `node_modules` during cleanup phase

### How was it tested?
- [x] Local
- [x] Jenkins

### CODE REVIEW

File: services/core/shared/core/transactionStatistics.js [hunk 1]
```diff
@@ -125,7 +125,10 @@ const insertToDb = async (statsList, date) => {
 
 	statsList.map(statistic => {
 		Object.assign(statistic, { date, amount_range: statistic.range });
-		statistic.id = String(statistic.date).concat('-').concat(statistic.amount_range);
+		statistic.id = String(statistic.date)
+			.concat('-').concat(statistic.type)
+			.concat('-')
+			.concat(statistic.amount_range);
```
Comment (sameersubudhi) [inline services/core/shared/core/transactionStatistics.js:131]:
```suggestion
			.concat('-').concat(statistic.amount_range);
```

File: Jenkinsfile [hunk 2]
```diff
@@ -140,6 +140,14 @@ pipeline {
...
+			dir('./framework') { sh 'rm -rf node_modules' }
+			dir('./services/core') { sh 'rm -rf node_modules' }
+			dir('./services/market') { sh 'rm -rf node_modules' }
+			dir('./services/gateway') { sh 'rm -rf node_modules' }
+			dir('./services/template') { sh 'rm -rf node_modules' }
+			dir('./tests') { sh "rm -rf node_modules" }
+
 			dir('./jenkins/lisk-core') { sh "make down" }
 			dir('./jenkins/mysql') { sh "make down" }
 			dir('./jenkins/redis') { sh "make down" }
```
Comment (sameersubudhi) [inline Jenkinsfile:153]: Kindly consider performing the cleanup after all the processes are down.
```suggestion
			dir('./jenkins/lisk-core') { sh "make down" }
			dir('./jenkins/mysql') { sh "make down" }
			dir('./jenkins/redis') { sh "make down" }
			
			dir('./') { sh 'rm -rf node_modules' }
			dir('./framework') { sh 'rm -rf node_modules' }
			dir('./services/core') { sh 'rm -rf node_modules' }
			dir('./services/market') { sh 'rm -rf node_modules' }
			dir('./services/gateway') { sh 'rm -rf node_modules' }
			dir('./services/template') { sh 'rm -rf node_modules' }
			dir('./tests') { sh "rm -rf node_modules" }
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]
Kindly consider the following changes.

File: Jenkinsfile [hunk 3]
```diff
@@ -140,6 +140,14 @@ pipeline {
 				sh 'pm2 stop --silent ecosystem.jenkins.config.js'
 			}
 
+			dir('./') { sh 'rm -rf node_modules' }
+			dir('./framework') { sh 'rm -rf node_modules' }
+			dir('./services/core') { sh 'rm -rf node_modules' }
+			dir('./services/market') { sh 'rm -rf node_modules' }
+			dir('./services/gateway') { sh 'rm -rf node_modules' }
+			dir('./services/template') { sh 'rm -rf node_modules' }
+			dir('./tests') { sh "rm -rf node_modules" }
```
Comment (MichalTuleja) [inline Jenkinsfile:149]: `make clean`?

File: services/core/shared/core/transactionStatistics.js (hunk 1 above)
Comment (MichalTuleja) [inline services/core/shared/core/transactionStatistics.js:131]:
```suggestion
		statistic.id = `${statistic.date}-${statistic.type}-${statistic.amount_range}`;
```

Comment (MichalTuleja) [reply services/core/shared/core/transactionStatistics.js:131]: btw, `concat` takes multiple params as well.

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: services/core/shared/core/transactionStatistics.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/core/shared/core/transactionStatistics.js]: I updated `concat` to take multiple params

```
statistic.id = String(statistic.date)
		        .concat('-', statistic.type)
			.concat('-', statistic.amount_range);
```

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**